website/docs/components/stepper/list/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Stepper List
+description: >-
+  A visual indicator of a linear vertical progress, where users can see where they are in a given flow.
```

Comment:
```suggestion
  Lists sequential steps for a user to complete.
```

Reply:
With the current text it's difficult for me to comprehend how this would be used and how it differs from the `Stepper::Nav` so I tried to explain it more simply and incorporate the main defining word for what it is, a "list".